config/kubefedconfig.yaml

```
@@ -26,4 +26,7 @@ spec:
     successThreshold: 1
     timeout: 3s
   syncController:
+    maxConcurrentReconciles: 1
     adoptResources: Enabled
```

nit: again personally prefer this to be sorted alphabetically but not a blocker.
```suggestion
    adoptResources: Enabled
    maxConcurrentReconciles: 1
```

---

Same as above.